BootloaderCommonPkg/Library/ShellLib/CmdBoot.c

```diff
@@ -503,7 +498,6 @@ ShellCommandBootFunc (
       ShellPrint (L"Enter index to change to (0 to %u)\n",
                   BootOptionList->OsBootOptionCount - 1
                   );
-      ShellPrint (L"(current index %u) ", mCurrentBoot);
```

Comment:
this print is useful. why remove ?

Reply:
This print is kind of in the spirit of how fdisk/gdisk will print the default values that you can just hit enter on to leave it as the default value. Maybe the text was not clear as "(current index)" but I agree it would be nice to have something display the current value in case the user did not really want to do 'c' option after realizing the current value.

Reply:
I was even thinking maybe we could add this to the OS Loader boot option printing, it made it easier to know which boot option would be tried next.

Reply:
Same as above. Will be kept in coming push.